pbqt.so/src/action.cpp

```diff
@@ -508,9 +512,13 @@ int pbqt_action::create_threads() {
       // perform peer check
       if (is_peer(gpu_id[i], gpu_id[j])) {
         RVSTRACE_
+
+        snprintf(srcgpuid_buff, sizeof(srcgpuid_buff), "%5d", gpu_id[i]);
```

Comment:
see if you can use std::setw() instead of %5d ?

Reply:
Yes, I tried it. But we need addition steps made the code kind of complex. This implementation easily fit into the code (especially **_msg_** log string) we have in place.